x-pack/plugins/security_solution/public/management/components/endpoint_responder/kill_process_action.tsx

```diff
@@ -85,6 +89,26 @@ export const KillProcessActionResult = memo<
     return <ResultComponent showAs="pending" />;
   }
 
+  // Show API errors if perform action fails
+  if (isError && apiError) {
+    return (
+      <ResultComponent
+        showAs="failure"
+        title={i18n.translate(
+          'xpack.securitySolution.endpointResponseActions.killProcess.performApiErrorMessageTitle',
+          { defaultMessage: 'Kill process action failed' }
+        )}
```

Comment:
I recently changed several of the success/error usages and removed the `title` so that the default/standard message is shown (ex. see the `<ActionError>` component. Unclear if this one should be removed as well or if it needs a custom error title

Reply:
I remember the UX discussion to have the messages be standard.  I'll do the same here.